### DISCUSSION TITLE
SUN-10K-SG04LP3-EU

### DISCUSSION THREAD

**Discussion by Bananebier** [StephanJoubert/home_assistant_solarman#170] [Q&A]
does it work for deye SUN-10K-SG04LP3-EU?

**Comment by Bananebier** [comment]
hello i watched the documentary and imitated it exactly despite the war i got no connection.
in your documentation it says because of the device serial number it means because it is 17xx and 21xx (protocol V5) but for me it starts with 23xxx

**Reply by Bananebier** [reply]
What can I do to make it work for me too?

**Reply by Bananebier** [reply]
all entities are somehow unavailable

**Reply by lukasniemeier** [reply]
You may try #171 which was based off the information found in #88

**Comment by MAPSOFT2** [comment]
I have a SUN-10K-SG04LP3-EU and my logger serial number starts with "23" as well ....
and it works with Home Assistant 2023.2.3 :-)

**Reply by tmbreuker** [reply]
I have a SUN-10K-SG04LP3-EU and my serial number starts with 24. However, my readings of SoC are not correct, consistently at 1%. Any ideas how I can fix this?

**Reply by tmbreuker** [reply]
switch to [davidrapan/ha-solarman](https://github.com/davidrapan/ha-solarman) sorted my issue
